package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@boardgame.io/redis-pubsub",
+  "private": false,
+  "version": "0.0.1",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "description": "Redis pub/sub adapter for boardgame.io",
+  "scripts": {
+    "prepare": "npm run build",
+    "prepublishOnly": "npm test && npm run lint",
+    "build": "tsc",
+    "test": "jest",
+    "fix": "prettier --write \"src/**/*.ts\"",
+    "lint": "tslint -p tsconfig.json"
+  },
+  "jest": {
+    "testMatch": [
+      "<rootDir>/src/**/*.test.ts"
+    ]
+  },
+  "funding": [
+    "https://github.com/boardgameio/boardgame.io?sponsor=1",
+    {
+      "type": "opencollective",
+      "url": "https://opencollective.com/boardgameio"
+    }
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/boardgameio/redis-pubsub.git"
+  },
+  "files": [
+    "lib/**/*"
+  ],
+  "keywords": [
+    "boardgame.io",
+    "redis"
+  ],
+  "author": "",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/boardgameio/redis-pubsub/issues"
+  },
+  "homepage": "https://github.com/boardgameio/redis-pubsub#readme",
+  "devDependencies": {
+    "@babel/core": "^7.15.0",
+    "@babel/preset-env": "^7.15.0",
+    "@babel/preset-typescript": "^7.15.0",
+    "@types/jest": "^27.0.1",
+    "@types/redis": "^2.8.31",
+    "babel-jest": "^27.0.6",
+    "jest": "^27.0.6",
+    "prettier": "^2.3.2",
+    "ts-mockito": "^2.6.1",
+    "tslint": "^6.1.3",
+    "tslint-config-prettier": "^1.18.0",
+    "typescript": "^4.3.5",
+    "@types/koa-router": "^7.4.4",
```

Comment:
You could use boardgame.io@0.47.3 in tests and drop this dependency now if you like, but we can also do that later.

Reply:
Done